<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimerSettingsWidget</class>
 <widget class="QWidget" name="TimerSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>195</width>
    <height>118</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Timer settings</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="fixedInterval">
       <property name="text">
        <string>Fixed interval</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QSpinBox" name="interval">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="suffix">
          <string>min</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="value">
          <number>15</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QRadioButton" name="randomInterval">
       <property name="text">
        <string>Random interval</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>From:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="randomFrom">
         <property name="suffix">
          <string>min</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>To:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="randomTo">
         <property name="suffix">
          <string>min</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="value">
          <number>30</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>fixedInterval</sender>
   <signal>toggled(bool)</signal>
   <receiver>interval</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>42</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>randomInterval</sender>
   <signal>toggled(bool)</signal>
   <receiver>label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>27</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>28</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>randomInterval</sender>
   <signal>toggled(bool)</signal>
   <receiver>randomFrom</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>93</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>randomInterval</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>randomInterval</sender>
   <signal>toggled(bool)</signal>
   <receiver>randomTo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
